### PR TITLE
Applications of artificial intelligence (AI) in design systems

### LINKED ISSUE #1084: Applications of artificial intelligence (AI) in design technologies
### Brief Summary: 
Describe the what, why, and how of your content idea in 2-5 sentences.

Artificial intelligence has enormously changed the engineering field. We are not in the age where engineering design was carried on using paper and a pencil.  
This article will outline some of the basic techniques engineers use today in design and how they are related to technology. 

### Key Takeaways: 
What are the 3-5 most important things the reader should understand or be able to do after reading this article?

1.	How has the design process changed due to technology evolution?
2.	Various applications of AI in the design process, with real examples.
3.	The future of AI in design.

### References: 
Please list links to any published content/research that you intend to use to support/guide this article. (If none, please indicate 'N/A'.)

### PR DESCRIPTION
**Pull request by Pius1991**
This fixes #1084

### Article Checklist
**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process, quality, and style-once published, you can then submit any number of articles.)**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by Bayler** [comment]
@lalith1403 can you allow me to perform a peer review test on this PR?

**Comment by lalith1403** [comment]
> @lalith1403 can you allow me to perform a peer review test on this PR?

Hey, I had performed changes on the PRs I had assigned over the weekend. I wasn't able to commit them due to the repository changes that took place recently. Pushing them now. :)

File: articles/applications-of-artificial-intelligence-(ai)-in-design-systems/index.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+#### Significant tasks performed during the design process.
+
+- Initialize design definition – It is the planning stage; the designer identifies and determines the technologies to be used on the project.
+- Design Characteristics – It is the stage for defining each system element's architecture and characteristics; also, the definition of the viability.
+- Alternatives for obtaining system elements
+- Managing the design
+
+### [Artificial Intelligence (AI)](https://www.section.io/engineering-education/artificial-intelligence-future/)
+
+It is human intelligence in machines that are programmed to think like a machine like a human and their actions. A computer uses machine learning to determine how they respond to these situations.
```
Comment (lalith1403) [inline articles/applications-of-artificial-intelligence-(ai)-in-design-systems/index.md:32]: This definition is erroneous. AI is not human intelligence. Be very careful with the definitions you give.

File: articles/applications-of-artificial-intelligence-(ai)-in-design-systems/index.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+### Real applications of AI in design systems
+
+#### Website development
+
+The use of tools like Tobii Pro Sprint, web-based eye-tracking technology, is used in developing websites. It allows UX designers to record website users' eye movements, thereby assisting in improving website design.
+
+The grid is the first company to offer the website design within minutes. The websites are custom AI-designed. After The Grid, many other companies have emerged with the same AI-designed websites, such as [Wix](https://www.wix.com/), and [Squarespace](https://www.squarespace.com/).
```
Comment (lalith1403) [inline articles/applications-of-artificial-intelligence-(ai)-in-design-systems/index.md:40]: grid -> Grid... since it is a proper noun.

Comment (lalith1403) [inline articles/applications-of-artificial-intelligence-(ai)-in-design-systems/index.md:40]: After Grid

File: articles/applications-of-artificial-intelligence-(ai)-in-design-systems/index.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+With AI technology, one only needs to upload the data content; and build the website from the ground within minutes.
+
+The same technology is used in design logos by the [Turbologo](https://turbologo.com/?ref=hackernoon.com). One need only upload the brand name, colors, and other preferences, and the logo is generated within minutes.
+
+#### Automation of legwork
+
+Technology can easily automate most of the designer tasks. Using AI, tasks like cropping images and correcting colors can be easily be automated. [Adobe Sensei](https://www.adobe.com/experience-platform/intelligent-services.html?ref=hackernoon.com) is a design tool that uses intelligent features. It assists the designers to perform work more efficiently and faster. Adobe Sensei identifies patterns in the image, edit, or even creates a new scene.
```
Comment (lalith1403) [inline articles/applications-of-artificial-intelligence-(ai)-in-design-systems/index.md:48]: The last sentence is not clear. Also, what is legwork? Define it.

File: articles/applications-of-artificial-intelligence-(ai)-in-design-systems/index.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+Another example is Netflix, which uses [augmented intelligence](https://www.section.io/engineering-education/introduction-to-extended-reality/) system for artwork personalization. When the need arises to create multiple banners, designers only need to look at the robot's banners to accept or reject them.
```
Comment (lalith1403) [inline articles/applications-of-artificial-intelligence-(ai)-in-design-systems/index.md:50]:
```suggestion
Another example is Netflix, which uses [augmented intelligence](/engineering-education/introduction-to-extended-reality/) system for artwork personalization. When the need arises to create multiple banners, designers only need to look at the robot's banners to accept or reject them.
```

File: articles/applications-of-artificial-intelligence-(ai)-in-design-systems/index.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+It is a system that helps in the design process by aiding in suggestions such as changes in position, scale, and elements.
+
+It assists the website designers with suggestions to create near-perfect websites. It doesn't design the website on its own, though it improves the designer's efficiency. Using it reduces maintenance efforts and time taken to complete the task.
+
+#### [Font Joy](https://fontjoy.com/)
+
+It uses machine learning to generate font combinations. Assists designers in choosing the best font combination. A person needs just to set different fonts; once done, click generate button. The page will load and will generate a new font combination.
+
+### Benefits of using AI in design systems
```
Comment (lalith1403) [inline articles/applications-of-artificial-intelligence-(ai)-in-design-systems/index.md:62]: The title says Applications of AI in Design systems. What systems are we talking about here? We need to be more specific. I don't think these qualify as systems. They are software. @hectorkambow @Pius1991 What do you think about this?

File: articles/applications-of-artificial-intelligence-(ai)-in-design-systems/index.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+- Initialize design definition – It is the planning stage; the designer identifies and determines the technologies to be used on the project.
+- Design Characteristics – It is the stage for defining each system element's architecture and characteristics; also, the definition of the viability.
+- Alternatives for obtaining system elements
+- Managing the design
+
+### [Artificial Intelligence (AI)](https://www.section.io/engineering-education/artificial-intelligence-future/)
+
+It is human intelligence in machines that are programmed to think like a machine like a human and their actions. A computer uses machine learning to determine how they respond to these situations.
+
+### Real applications of AI in design systems
```
Comment (lalith1403) [inline articles/applications-of-artificial-intelligence-(ai)-in-design-systems/index.md:34]:
```suggestion
### Applications of AI in design systems
```

File: articles/applications-of-artificial-intelligence-(ai)-in-design-systems/index.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+
+The below are primary elements of a system;
+
+- Architecture – It is the backbone of a system, defines the structure, behavior, and view of a course.
+- Modules – Smallest units in a system, each is handling a specific task. Several modules together make a system.
+- Components – They are made up of modules; they provide a particular group function.
+- Interfaces – Boundaries upon which components share information and relate.
+- Data – A collection of facts and information.
+
+#### Significant tasks performed during the design process.
```
Comment (lalith1403) [inline articles/applications-of-artificial-intelligence-(ai)-in-design-systems/index.md:23]:
```suggestion
#### Significant design tasks 
```

**Review by lalith1403 — CHANGES_REQUESTED** [review]
Make the requested changes.

**Comment by hectorkambow** [comment]
plagiarism check done

**Comment by hectorkambow** [comment]
@lalith1403  
Just wanted to get a status report on this PR?

**Comment by lalith1403** [comment]
Waiting for the author to make changes. @hectorkambow

**Comment by hectorkambow** [comment]
👍  Author pinged via slack

File: articles/applications-of-artificial-intelligence-(ai)-in-design-systems/index.md (hunk 5 above)
Comment (Pius1991) [reply articles/applications-of-artificial-intelligence-(ai)-in-design-systems/index.md:62]: > The title says Applications of AI in Design systems. What systems are we talking about here? We need to be more specific. I don't think these qualify as systems. They are software. @hectorkambow @Pius1991 What do you think about this?

@lalith1403 I think "Applications of AI in Design Technologies" would sound better.

**Comment by Pius1991** [comment]
@lalith1403 I have made the requested changes.

**Comment by hectorkambow** [comment]
@Pius1991  Please go over the comments and edits made by your peer reviewer and myself. 

I found the article lacking in the some areas - specifically the intro - it took me a while to understand what you were trying to discuss in your article. 

You should be direct and to the point - explain in the intro what you will be discussing with your readers. 

Future articles will not be allowed much revisions moving forward - so keen attention on how to provide the adequte substance and information in your article will be key 🚀  

cc @lalith1403  please go over the fixes as well (especially with the front matter)

**Pull request merged**